Dockerfile

```diff
@@ -18,13 +17,9 @@ RUN apt-get install -y ros-$ROS_DISTRO-cv-camera \
 
 COPY . .
```

Comment:
Shouldn't you do something like COPY . ~/Programming/

Reply:
Further up there is a line that sets the working directory to /home/spaceuser/Programming/robotics-prototype (`WORKDIR /home/spaceuser/Programming/robotics-prototype`). Like I said earlier just make sure you `sudo docker-compose up --build` to take changes into account